tests/test_deserialization.py

```diff
@@ -1338,6 +1338,50 @@ class AliasingUserSerializer(Schema):
         assert result["name"] == "Mick"
         assert result["years"] is None
 
+    # https://github.com/marshmallow-code/marshmallow/issues/713
+    @pytest.mark.parametrize(
+        ("missing", "missing_values", "input_data", "expected"),
+        [
+            (None, {None}, {"name": None}, {"name": None}),
+            (None, {None}, {"name": ""}, {"name": ""}),
+            (None, {""}, {"name": ""}, {"name": None}),
+            ("", {""}, {"name": ""}, {"name": ""}),
+            ("", {None}, {"name": None}, {"name": ""}),
+        ],
+    )
+    def test_deserialize_with_custom_missing_values(
+        self, missing, missing_values, input_data, expected
+    ):
+        class ArtistSchema(Schema):
+            name = fields.String(missing=missing, missing_values=missing_values)
+
+        schema = ArtistSchema()
+        assert schema.load(input_data) == expected
+
+    def test_deserialize_required_field_with_custom_missing_values(self):
+        class ArtistSchema(Schema):
+            album_names = fields.List(
+                fields.Str(), required=True, missing_values=([], ())
+            )
+
+        with pytest.raises(ValidationError, match="required"):
+            ArtistSchema().load({"album_names": []})
+
+    def test_setting_default_missing_values(self, monkeypatch):
```

Comment:
This test might not be necessary--it's sorta just testing the Python language. Just wanted to make sure [this use case was met](https://github.com/marshmallow-code/marshmallow/issues/713#issuecomment-529205297).